### ISSUE TITLE
Rotate otobo.log

### ISSUE THREAD

**Issue by svenoe** [RotherOSS/otobo#3909]
We should rotate var/log/otobo.log, to not let it grow too much. Probably have a look at and align with the rotation in `bin/otobo.Daemon.pl->_LogFilesSet` in some way.

**Comment by stefanhaerter** [comment]
(Just a quick thought)
Perhaps we could make use of [man logrotate](https://linux.die.net/man/8/logrotate) and place it as a cron job file in `var/cron/`, this way it may be managed via `bin/Cron.sh`.

**Comment by bschmalhofer** [comment]
Log::Log4perl can also rotate logfiles. See #402.

**Comment by svenoe** [comment]
> Log::Log4perl can also rotate logfiles. See #402.

Not in principle against it, but I think this is a decision we should make independent of this issue, as the necessary work to adapt Kernel/System/Log.pm will be higher, and I would put it in rel-11_1, if we decide for it.

**Comment by StefanRother-OTOBO** [comment]
Yes, I think we need a fast solution for all otobo docker versions now, we found a view really big otobo.log files in the last days (10.1 and 11.0).

**Comment by MichaelThumes** [comment]
merged
